workflows/surgical_agent/surgical_agent.sh

```
@@ -0,0 +1,343 @@
+    chmod +x run-surgical-agents.sh
+
+    print_status "Running surgical agents docker services..."
+    ./run-surgical-agents.sh
+}
+
+# Function to stop video app
+stop_video_app() {
+    print_status "Stopping WebRTC video streaming containers..."
+
+    # Find containers by name (both running and stopped)
+    local containers=$(docker ps -aq --filter "name=^/${WEBRTC_CONTAINER_NAME}$")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)